src/modality-specific-files/physiological-recordings.md

```diff
@@ -72,122 +194,585 @@ A guide for using macros can be found at
+
+Eye-tracking files `<matches>_recording-<label>_physio.tsv.gz` MAY be annotated
+with a corresponding `<matches>_recording-<label>_physioevents.tsv.gz` file.
+The `<matches>_recording-<label>_physioevents.tsv.gz` file MAY be employed to
+record discontinuous model parameters generated by the eye-tracker, for example,
+those derived from the saccade and blinks model some eye-trackers produce.
+
+**Examples**.
+The recordings produced by a monocular eye-tracker during a
+visual search task may display the following structure:
+<!-- This block generates a file tree.
+A guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_filetree_example({
+  "sub-01": {
+    "func": {
+      "sub-01_task-visualSearch_bold.json": "",
+      "sub-01_task-visualSearch_bold.nii.gz": "",
+      "sub-01_task-visualSearch_events.json": "",
+      "sub-01_task-visualSearch_events.tsv": "",
+      "sub-01_task-visualSearch_recording-eye1_physio.json": "",
+      "sub-01_task-visualSearch_recording-eye1_physio.tsv.gz": "",
+      "sub-01_task-visualSearch_recording-eye1_physioevents.json": "",
+      "sub-01_task-visualSearch_recording-eye1_physioevents.tsv.gz": ""
+    },
+  },
+}) }}
+
+The above example is extended to a binocular eye-tracker producing three signals
+(left and right eyes, plus a *cyclopean* recording), as follows:
+<!-- This block generates a file tree.
+A guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_filetree_example({
+  "sub-01": {
+    "func": {
+      "sub-01_task-visualSearch_bold.json": "",
+      "sub-01_task-visualSearch_bold.nii.gz": "",
+      "sub-01_task-visualSearch_events.json": "",
+      "sub-01_task-visualSearch_events.tsv": "",
+      "sub-01_task-visualSearch_recording-eye1_physio.json": "",
+      "sub-01_task-visualSearch_recording-eye1_physio.tsv.gz": "",
+      "sub-01_task-visualSearch_recording-eye1_physioevents.json": "",
+      "sub-01_task-visualSearch_recording-eye1_physioevents.tsv.gz": "",
+      "sub-01_task-visualSearch_recording-eye2_physio.json": "",
+      "sub-01_task-visualSearch_recording-eye2_physio.tsv.gz": "",
+      "sub-01_task-visualSearch_recording-eye2_physioevents.json": "",
+      "sub-01_task-visualSearch_recording-eye2_physioevents.tsv.gz": "",
+      "sub-01_task-visualSearch_recording-eye3_physio.json": "",
+      "sub-01_task-visualSearch_recording-eye3_physio.tsv.gz": "",
+      "sub-01_task-visualSearch_recording-eye3_physioevents.json": "",
+      "sub-01_task-visualSearch_recording-eye3_physioevents.tsv.gz": ""
+    },
+  },
+}) }}
+
+Given the above example file structures, a corresponding
+`sub-01_task-visualSearch_recording-eye1_physio.json` sidecar
+could read:
+
+```JSON
+{
+    "DeviceSerialNumber": "17535483",
+    "Columns": ["timestamp", "x_coordinate", "y_coordinate", "pupil_size"],
```

Comment:
In the src/schema/objects/columns.yaml and src/schema/rules/tabular_data/physio.yaml it says "et_timestamp", here it says "timestamp", which is correct?

Reply:
Both are correct, these are two different things:

- `src/schema/objects/columns.yaml`, `src/schema/rules/tabular_data/physio.yaml` define a `et_timestamp` column name. This is so to allow the timestamp be different from the already existing ``timestamp`` column. This identifier is internal to the schema, so it does not propagate into actual column names.

- the line 672 you commented on is just the necessary specification of the `"Columns"` metadata, the column name is arbitrary, and in this case it is named `"timestamp"` but other names would be equally supported.